interactive_engine/compiler/src/main/java/com/alibaba/graphscope/common/ir/meta/schema/GraphOptTable.java

```diff
@@ -122,6 +122,9 @@ private RelDataType deriveType(GraphProperty property) {
                 return typeFactory.createSqlType(SqlTypeName.BIGINT);
             case DOUBLE:
                 return typeFactory.createSqlType(SqlTypeName.DOUBLE);
+            case DATE:
```

Comment:
It seems that there is a lack of handling for the cases of `DateTime` and `Time`？